### DISCUSSION TITLE
Show Word Count

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#784] [Feature Requests]
A user has requested the ability to see the word count of a note. While this is simple enough to calculate (in English), I'm not sure where to display this information.

Any suggestions are most welcome.

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by AnetteMM** [comment]
One possibility could be in a corner of the bottom as visible wordcount or in the ... menu as a "hidden" wordcount. I think there is no wrong or right way, as everbody has her/his own preferences; when it's hidden it's hidden, when it's visible it's visible. I like both :-)  👍

**Comment by hollispotter** [comment]
As a menu option while editing would be preferred here. Constantly seeing the count distracts me, but when I want to check, it would only be two taps away. Thanks for considering!
